src/main.js

```diff
@@ -49,6 +49,10 @@ function viewMakeOwnForm() {
   randomCoverButton.classList.add('hidden');
   saveCoverButton.classList.add('hidden');
   homeButton.classList.remove('hidden');
+  inputCoverImage.value = '';
+  inputTitle.value = '';
+  inputDesc1.value = '';
```

Comment:
I am so happy we thought about changing the inputs to an empty string after we left the cover form view.